scripts/rebuild_glyphs.py

```diff
@@ -0,0 +1,57 @@
+import json
+import os
+
+script_path = os.path.realpath(__file__)
+script_dir = os.path.abspath(os.path.join(script_path, os.pardir))
+
+cmd_path = os.path.abspath(os.path.join(script_dir, os.pardir, "node_modules", "markdown-pdf", "bin", "markdown-pdf"))
```

Comment:
Is PDF generation by this command idempotent? I ask because it's being run unconditionally on all glyphs, including ones that have not changed.  With the prior PDF conversion, that would generate diffs in every PDF every time, polluting commits.

Reply:
I've just pushed a change that looks at the mtime for the PDF and the source directory (containing the README and image files), and skips the PDF generation if the PDF was generated more recently.

The PDFs generated by this script get used in the automatic building of the complete specification PDF, but aren't (currently) automatically committed to the repository.

Reply:
What is the workflow for a manual update, then? I want to make sure that it's still possible to build the specification PDFs locally by hand as well, and that we aren't only dependent on GitHub actions.

Reply:
You should be able to run the `rebuild_glyphs.py` script as a drop-in replacement for the previous shell scripts. It's a python script with no python package dependencies (the only modules it imports are part of the standard library), and it relies on the same external program for the PDF generation (`markdown-pdf`). 

So:

```bash
cd scripts
npm install markdown-pdf
python3 rebuild_glyphs.py
```

should regenerate only the updated PDFs in `specifications/glyphscript/Glyphs/`, and update the `.tex` files in `specifications/glyphscript/`.

Reply:
Can we get this into the appropriate README file(s), so this documentation is easy to find?  After that, I'm ready to hit merge on the pull request.